src/components/Login.jsx

```diff
@@ -37,27 +39,34 @@ const Login = ({setIsAuthenticated}) => {
     setLoading(true);
     const response = await login({email, password});
     setLoading(false);
-    if (response.jwt) {
-      const duration = new Date();
-      duration.setTime(duration.getTime() + (1 * 60 * 60 * 1000));
-      cookies.set('token', response.jwt, {path: '/', expires: duration});
-      const dataSet = {
-        'id': response.user.id,
-        'name': response.user.name,
-        'isSubscribed': response.user.isSubscribed,
-      };
-      localStorage.setItem('data', JSON.stringify(dataSet));
-      setIsAuthenticated(true);
-      navigate('/home');
+    console.log(response);
```

Comment:
console logs are great for debugging, but generally, you don't want to commit them
```suggestion
```